src/doc/rustc/src/platform-support/nto-qnx.md

```diff
@@ -18,30 +20,29 @@ and [Blackberry QNX][BlackBerry].
 
 ## Requirements
 
-Currently, the following QNX Neutrino versions and compilation targets are supported:
+Currently, the following QNX versions and compilation targets are supported:
```

Comment:
Maybe the `target_os` should be migrated to `"qnx"`... (we don't require the target tuple and the `target_os` to line up perfectly)

Reply:
just a random thought right now, I won't blockade anything on that.

Reply:
Yes, I agree. Having `Neutrino` there is quite confusing, everybody I know just calls it `QNX`. Same for the target triplet, we could shorten it.
However, this would also mean that we become incompatible, and this would hurt a some people...